xknx/knxip/knxip.py

```diff
@@ -105,6 +105,10 @@ def __str__(self) -> str:
         """Return object as readable string."""
         return f'<KNXIPFrame {self.header}\n body="{self.body}" />'
 
+    def __repr__(self) -> str:
+        """Representation of object."""
+        return self.__str__()
```

Comment:
You can simply set `__repr__ = __str__`. No need to write another function.

Reply:
Thanks for the hint. After a little searching the web I realised that `object.str()` falls back to `repr(object)` if no `__str__` function is given, so we can just omit `__str__` to achieve the same. 1 line less 🤣